src/processor.ts

```diff
@@ -8,4 +8,16 @@ export class Processor {
   checkConsent(userInput: User): boolean {
     return userInput.consentGiven === true
   }
+
+  checkAge(userInput: User, age: number): void {
+    if (age >= 18) {
+      userInput.consentGiven = true
+    } else {
+      userInput.consentGiven = false
+    }
+  }
+
+  revokeConsent(userInput: User): void {
```

Comment:
If you intend to revoke consent, it might make more sense to set consentGiven to false

Reply:
resolved -> changed to false